README.md

```diff
@@ -120,6 +120,48 @@ Launch the application with the following commands:
 
 ![CloudWatchMetrics01.png](docs/images/BookstoreRVizPlan01.png)
 
+## Run with a WorldForge world
+
+After exporting a world from WorldForge, we can unzip the content and move under simulation_ws package:
+
+```bash
+unzip exported_world.zip
+mv aws_robomaker_worldforge_pkgs simulation_ws/src/
+```
+
+Build it again
+
+```bash
+cd simulation_ws
+colcon build
+```
+
+Launch the application with the following commands:
+
+```bash
+source simulation_ws/install/local_setup.sh
+roslaunch cloudwatch_simulation worldforge_turtlebot_navigation.launch
+```
+
+By default, WorldForge packages will load the exported world. To override, specify the environment variable `WORLD_ID`. 
+
+```bash
+# use worldId found in "src/aws_robomaker_worldforge_worlds/worlds"
+# e.g, generation_05wq8sybdcn2_world_1
+export WORLD_ID=<worldId>  
+```
+
+### Navigation in a WorldForge world
```

Comment:
what's the difference between "Run with a Worldforge world" vs "Navigation in a WorldForge world"?

Reply:
Since the map generation plugin is optional, I separate them on purpose.